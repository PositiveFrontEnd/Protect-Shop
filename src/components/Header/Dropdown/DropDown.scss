.dropdown {
  width: 316px;
  text-transform: uppercase;
}

.dropdown__wrapper {
  position: fixed;
  top: 0;
  left: 0;
  z-index: 50;
  width: 100%;
  height: 100%;
  background: rgb(0 0 0 / 30%);

  @media (width >= 768px) {
    display: none;
  }
}

.dropdown__top {
  display: flex;
  align-items: center;
  justify-content: space-between;
  padding: 38px 24px 16px;
  background: var(--black);

  .dropdown__title {
    color: var(--white);
    font-weight: 700;
  }
}

.dropdown__search {
  display: flex;
  gap: 8px;
  align-items: center;
  justify-content: start;
  padding: 16px 20px 28px;
  background: #f8f8fa;

  .dropdown__search__title {
    width: 220px;
    font-size: 18px;
    line-height: 23px;
    background: #f8f8fa;
    border-bottom: 1px solid #e7e7e8;

    &::placeholder {
      font-size: 24px;
      line-height: 33px;
    }
  }
}

.dropdown__menu {
  display: flex;
  flex-direction: column;

  .dropdown__list {
    padding: 13px 0 13px 24px;
    color: var(--black);
    font-weight: 600;
    line-height: 22px;
    background: var(--white);
    border-top: 1px solid #e7e7e8;
    position: relative;
    &::after {
      position: absolute;
      right: 24px;
      bottom: 18px;
      width: 7px;
      height: 12px;
      background: url("../HeaderImgs/arrowR.png") no-repeat;
      background-size: cover;
      content: "";
    }
  }

  .active {
    padding: 13px 0 13px 24px;
    color: var(--incorrect);
    font-weight: 600;
    line-height: 22px;
    background: var(--white);
    border-top: 1px solid #e7e7e8;
  }
}

.dropdown__contacts {
  display: flex;
  flex-direction: column;
  gap: 20px;
  height: 230px;
  padding: 28px 0 0 24px;
  background: #f8f8fa;

  .dropdown__contacts__item {
    display: flex;
    gap: 8px;
    align-items: center;
    justify-content: start;
  }

  .dropdown__contacts__text {
    font-size: 14px;
    line-height: 19px;
  }
}
