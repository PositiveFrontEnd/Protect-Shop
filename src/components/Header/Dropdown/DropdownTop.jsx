import React from 'react';
import Close from '../HeaderSvg/close.svg?react'
import PropTypes from "prop-types";

const DropdownTop = ({ click }) => {

    return (
        <div className='dropdown__top'>
            <div></div>
            <h4 className='dropdown__title' onClick={click}>close</h4>
            <Close onClick={click} />
        </div>
    );
};

DropdownTop.propTypes = {
    click: PropTypes.func,
}

export default DropdownTop;