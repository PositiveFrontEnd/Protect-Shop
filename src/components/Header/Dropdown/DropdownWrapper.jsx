import React from 'react';
import PropTypes from "prop-types";

const DropdownWrapper = ({click, children}) => {
    return (
        <div className='dropdown__wrapper' onClick={click}>{children}
        </div>
    );
};
DropdownWrapper.propTypes = {
    click: PropTypes.func,
    children:PropTypes.any
}
export default DropdownWrapper;