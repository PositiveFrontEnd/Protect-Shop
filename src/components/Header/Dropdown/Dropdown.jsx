import React from 'react';
import DropdownTop from "./DropdownTop";
import PropTypes from "prop-types";
import "./DropDown.scss"
import SearchB from "src/components/Header/HeaderSvg/searchB.svg?react";
import {NavLink, Link, useNavigate} from "react-router-dom";
import PhoneIncomingB from "src/components/Header/HeaderSvg/phoneIncomingB.svg?react";
import PhoneB from "src/components/Header/HeaderSvg/phoneB.svg?react";
import Time from "src/components/Header/HeaderSvg/time.svg?react";
import ModalForm from "src/components/Modal/ModalForm";
import {useDispatch, useSelector} from "react-redux";
import {actionInputText} from "src/store/homeSlice";
import {
    actionSearchProducts
} from "src/store/productsSlice";
import {selectInputText} from "src/store/selectors";

const Dropdown = ({actionDropdown, modalChange, isModal}) => {

    const dispatch = useDispatch();
    const navigate = useNavigate();
    const isSale = location.search.includes('status=sell');
    const isNew = location.search.includes('status=new');

    const inputText = useSelector(selectInputText);
    const handleInputChange = (event) => {
        dispatch(actionInputText(event.target.value));
    };

    const stopClick = (event) => {
        event.stopPropagation()
    };

    const handleSearch = () => {
        dispatch(actionSearchProducts({query: `${inputText}`}))
        navigate(`/search?request=${encodeURIComponent(inputText)}`)
    }

    const handleEnterPress = (event) => {
        if (event.key === 'Enter') {
            return handleInputChange(event);
        }
    };

    const handleSubmit = (e) => {
        e.preventDefault();
        actionDropdown();
        handleSearch();
    };

    return (
        <div className='dropdown' onClick={stopClick}>
            <DropdownTop click={actionDropdown}/>
            <div className='dropdown__search'>
                <SearchB onClick={() => {
                    actionDropdown()
                    handleSearch()
                }}/>
                <form onSubmit={handleSubmit}>
                    <input className='dropdown__search__title'
                           type='text'
                           name='search'
                           placeholder='SEARCH'
                           value={inputText}
                           onKeyDown={handleEnterPress}
                           onChange={handleInputChange}/>
                </form>
            </div>
            <nav className='dropdown__menu'>
                <Link onClick={actionDropdown}
                      className={isSale ? 'dropdown__list active' : 'dropdown__list'}
                      to='/catalogue?status=sell'>sale</Link>
                <Link onClick={actionDropdown}
                      className={isNew ? 'dropdown__list active' : 'dropdown__list'}
                      to='/catalogue?status=new'>new</Link>
                <NavLink onClick={actionDropdown}
                         className='dropdown__list'
                         to='/catalogue'>catalogue</NavLink>
                <NavLink onClick={actionDropdown}
                         className='dropdown__list'
                         to='/shops'>shops</NavLink>
                <NavLink onClick={actionDropdown}
                         className='dropdown__list'
                         to='/about-us'>about us</NavLink>
            </nav>
            <div className='dropdown__contacts'>
                <div className='dropdown__contacts__item' onClick={modalChange}>
                    <PhoneIncomingB/><p className='dropdown__contacts__text' onClick={actionDropdown}>Call me out</p>
                </div>
                <div className='dropdown__contacts__item'>
                    <PhoneB/><a className='dropdown__contacts__text' href="tel:+380505000096">+380(50) 500 00 96</a>
                </div>
                <div className='dropdown__contacts__item'>
                    <Time/><p className='dropdown__contacts__text'>Mon-Sun: 11:00 â€“ 20:00</p>
                </div>
            </div>
            {
                isModal && (<ModalForm isOpen={modalChange} onclick={modalChange}/>)
            }
        </div>
    )
        ;
};

Dropdown.propTypes = {
    actionDropdown: PropTypes.func,
    modalChange: PropTypes.func,
    isModal: PropTypes.bool
}

export default Dropdown;