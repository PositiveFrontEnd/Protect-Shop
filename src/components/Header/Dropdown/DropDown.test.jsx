import React from "react";
import { render } from "@testing-library/react";
import { Provider } from "react-redux";
import Dropdown from "./Dropdown";
import { BrowserRouter } from "react-router-dom";
import configureStore from "redux-mock-store";

describe("Тестування компонента  DropdownTop", () => {
    test("Знімок", () => {
        const mockStore = configureStore();
        const initialState = {
            home: {
                isAnimation: false,
                inputText: "some text",
                filterHomePage: [],
            },
            user: {
                token: "token",
                registrationModal: false,
                authorizationStatus: null,
                registrationStatus: "some text",
                registrationData: {},
                correctData: "some text",
                isAdmin: false,
            },
            products: {
                products: [],
                productsByCategory: [],
                productsByType: [],
                oneProduct: {},
                ThreeProducts: [],
                ThreeColors: [],
                filters: [],
                twelveProducts: [],
                filtersParameters: {},
                twelveFiltersProducts: [],
                searchProducts: [],
                youSee: [],
                previewProductInfo: {},
            },
            order: {
                orderState: [],
                productsForOrderGuest: {},
                infoForOrderGuest: [],
                deliveryForOrderGuest: {},
                order: {
                    letterSubject: "Thank you for order!",
                },
                promoCodePrice: "",
                orderGuest: { letterSubject: "Thank you for order!" },
            },
            basket: {
                basket: [],
                priceGuest: "''",
                guestBasket: {},
            },
            catalogue: {
                sortCatalogue: "some text",
                loadFilter: false,
            },
            favorite: {
                favorite: [],
                favoriteForCustomer: [],
                guestFavorite: [],
            },
            message: {
                letters: [],
            },
        };
        const data = {
            isModal: true,
            modalChange: () => console.log("hello"),
            actionDropdown: () => console.log("hello"),
        };

        const store = mockStore(initialState);
        const dropdown = render(
            <Provider store={store}>
                <BrowserRouter>
                    <Dropdown data={data} />
                </BrowserRouter>
            </Provider>
        );

        expect(dropdown).toMatchSnapshot();
    });
});
