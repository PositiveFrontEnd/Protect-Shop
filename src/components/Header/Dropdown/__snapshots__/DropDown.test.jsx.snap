// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Тестування компонента  DropdownTop > Знімок 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="dropdown"
      >
        <div
          class="dropdown__top"
        >
          <div />
          <h4
            class="dropdown__title"
          >
            close
          </h4>
          <svg
            fill="none"
            height="16"
            viewBox="0 0 16 16"
            width="16"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M7.12587 8L2 2.87413L2.87413 2L8 7.12587L13.1259 2L14 2.87413L8.87413 8L14 13.1259L13.1259 14L8 8.87413L2.87413 14L2 13.1259L7.12587 8Z"
              fill="white"
              fill-rule="evenodd"
            />
          </svg>
        </div>
        <div
          class="dropdown__search"
        >
          <svg
            fill="none"
            height="33"
            viewBox="0 0 32 33"
            width="32"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M22.1553 24.5414L27.8647 30.2521L29.7513 28.3654L24.0407 22.6561C25.7436 20.5317 26.6699 17.8894 26.666 15.1667C26.666 8.54275 21.29 3.16675 14.666 3.16675C8.04202 3.16675 2.66602 8.54275 2.66602 15.1667C2.66602 21.7907 8.04202 27.1667 14.666 27.1667C17.3887 27.1706 20.031 26.2444 22.1553 24.5414ZM21.166 21.8667C19.4259 23.5589 17.0933 24.5039 14.666 24.5001C9.50868 24.5001 5.33268 20.3227 5.33268 15.1667C5.33268 10.0094 9.50868 5.83341 14.666 5.83341C19.822 5.83341 23.9993 10.0094 23.9993 15.1667C24.0032 17.594 23.0582 19.9266 21.366 21.6667L21.166 21.8667Z"
              fill="#191919"
            />
          </svg>
          <form>
            <input
              class="dropdown__search__title"
              name="search"
              placeholder="SEARCH"
              type="text"
              value=""
            />
          </form>
        </div>
        <nav
          class="dropdown__menu"
        >
          <a
            class="dropdown__list"
            href="/catalogue?status=sell"
          >
            sale
          </a>
          <a
            class="dropdown__list"
            href="/catalogue?status=new"
          >
            new
          </a>
          <a
            class="dropdown__list"
            href="/catalogue"
          >
            catalogue
          </a>
          <a
            class="dropdown__list"
            href="/shops/kharkiv"
          >
            shops
          </a>
          <a
            class="dropdown__list"
            href="/about-us"
          >
            about us
          </a>
        </nav>
        <div
          class="dropdown__contacts"
        >
          <div
            class="dropdown__contacts__item"
          >
            <svg
              fill="none"
              height="17"
              viewBox="0 0 16 17"
              width="16"
              xmlns="http://www.w3.org/2000/svg"
            >
              <g
                clip-path="url(#clip0_511_22277)"
              >
                <path
                  d="M15.5261 2.3856C15.7864 2.12525 15.7864 1.70314 15.5261 1.44279L15.0547 0.971385C14.7943 0.711035 14.3722 0.711036 14.1119 0.971385L11.2834 3.79983L10.1049 2.62127C9.9142 2.43061 9.62746 2.37357 9.37833 2.47676C9.1292 2.57995 8.9668 2.82303 8.9668 3.09268V6.86391C8.9668 7.23213 9.26526 7.5306 9.63346 7.5306H13.4047C13.6743 7.5306 13.9175 7.36813 14.0206 7.11903C14.1238 6.86992 14.0668 6.58317 13.8761 6.39251L12.6977 5.21404L15.5261 2.3856Z"
                  fill="#191919"
                />
                <path
                  d="M1.33791 3.52769C1.27065 2.57483 2.07752 1.83325 3.03079 1.83325H5.07135C5.56069 1.83325 6.00203 2.12809 6.19023 2.58072L7.04351 4.63289C7.25138 5.1328 7.10385 5.71021 6.68185 6.04853L6.14994 6.47495C5.91257 6.66524 5.83187 6.99338 5.97261 7.26307C6.69591 8.64934 7.76491 9.82581 9.06491 10.6777C9.33931 10.8575 9.70178 10.786 9.90651 10.5295L10.4608 9.83527C10.7984 9.41241 11.3746 9.26454 11.8734 9.47287L13.9214 10.3279C14.3731 10.5165 14.6673 10.9588 14.6673 11.4491V12.1577H14.6399V13.4665C14.6399 14.4217 13.8999 15.2303 12.949 15.1626C6.86871 14.7301 1.76896 9.63441 1.33791 3.52769Z"
                  fill="#191919"
                />
              </g>
              <defs>
                <clippath
                  id="clip0_511_22277"
                >
                  <rect
                    fill="white"
                    height="16"
                    transform="translate(0 0.5)"
                    width="16"
                  />
                </clippath>
              </defs>
            </svg>
            <p
              class="dropdown__contacts__text"
            >
              Call me out
            </p>
          </div>
          <div
            class="dropdown__contacts__item"
          >
            <svg
              fill="none"
              height="17"
              viewBox="0 0 16 17"
              width="16"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M1.33791 3.52769C1.27065 2.57483 2.07752 1.83325 3.03079 1.83325H5.07135C5.56069 1.83325 6.00203 2.12809 6.19023 2.58072L7.04351 4.63289C7.25138 5.1328 7.10385 5.71021 6.68185 6.04853L6.14994 6.47495C5.91257 6.66524 5.83187 6.99338 5.97261 7.26307C6.69591 8.64934 7.76491 9.82581 9.06491 10.6777C9.33931 10.8575 9.70178 10.786 9.90651 10.5295L10.4608 9.83527C10.7984 9.41241 11.3746 9.26454 11.8734 9.47287L13.9214 10.3279C14.3731 10.5165 14.6673 10.9588 14.6673 11.4491V12.1577H14.6399V13.4665C14.6399 14.4217 13.8999 15.2303 12.949 15.1626C6.86871 14.7301 1.76896 9.63441 1.33791 3.52769Z"
                fill="#191919"
              />
            </svg>
            <a
              class="dropdown__contacts__text"
              href="tel:+380505000096"
            >
              +380(50) 500 00 96
            </a>
          </div>
          <div
            class="dropdown__contacts__item"
          >
            <svg
              fill="none"
              height="17"
              viewBox="0 0 16 17"
              width="16"
              xmlns="http://www.w3.org/2000/svg"
            >
              <g
                clip-path="url(#clip0_511_22270)"
              >
                <path
                  d="M13.9045 6.25251C14.1618 6.25251 14.4036 6.15238 14.5855 5.97049C14.7674 5.78856 14.8676 5.54677 14.8676 5.28952C14.8676 5.03227 14.7674 4.79048 14.5855 4.60859L13.6322 3.65529C13.4503 3.47336 13.2085 3.37323 12.9513 3.37323C12.694 3.37323 12.4522 3.47336 12.2703 3.65529C11.9251 4.0005 11.8976 4.54469 12.1872 4.92196L11.9604 5.13448C11.1502 4.48702 10.1796 4.03234 9.11842 3.84118V3.35659C9.88137 3.328 10.4933 2.69877 10.4933 1.92898C10.4933 1.14103 9.85229 0.5 9.06435 0.5H6.90449C6.11655 0.5 5.47552 1.14103 5.47552 1.92898C5.47552 2.69877 6.08747 3.328 6.85042 3.35659V3.84118C5.77996 4.03404 4.8017 4.49504 3.98725 5.15154L3.79205 4.94874C4.10406 4.57086 4.08345 4.00878 3.73003 3.65529C3.54814 3.47336 3.30628 3.37323 3.04907 3.37323C2.79185 3.37323 2.55003 3.47336 2.36814 3.65529L1.41487 4.60859C1.23298 4.79048 1.13281 5.03227 1.13281 5.28952C1.13281 5.54677 1.23298 5.78856 1.41487 5.97045C1.59676 6.15238 1.83862 6.25251 2.09584 6.25251C2.31346 6.25251 2.51997 6.18055 2.68858 6.04831L2.90129 6.26934C2.08797 7.34025 1.60451 8.67472 1.60451 10.1201C1.60451 13.638 4.46655 16.5 7.98444 16.5C11.5023 16.5 14.3643 13.638 14.3643 10.1201C14.3643 8.66466 13.8741 7.32172 13.0505 6.24703L13.2853 6.02691C13.4586 6.17287 13.6754 6.25251 13.9045 6.25251ZM13.2299 10.1201C13.2299 13.0124 10.8768 15.3656 7.98444 15.3656C5.09208 15.3656 2.73895 13.0124 2.73895 10.1201C2.73895 7.22775 5.09208 4.87462 7.98444 4.87462C10.8768 4.87466 13.2299 7.22775 13.2299 10.1201Z"
                  fill="#191919"
                />
                <path
                  d="M7.98418 6.07544C5.75391 6.07544 3.93945 7.8899 3.93945 10.1202C3.93945 12.3504 5.75391 14.1649 7.98418 14.1649C10.2144 14.1649 12.0289 12.3504 12.0289 10.1202C12.0289 7.8899 10.2144 6.07544 7.98418 6.07544ZM10.1333 8.99204L8.79761 10.1618C8.78774 10.3564 8.70939 10.5482 8.56078 10.6968C8.24234 11.0152 7.72602 11.0152 7.40758 10.6968C7.08914 10.3783 7.08914 9.862 7.40758 9.54357C7.58376 9.36739 7.82044 9.28911 8.05073 9.30791L9.38585 8.13864C9.62155 7.93229 9.97984 7.95596 10.1863 8.19162C10.3927 8.42732 10.3689 8.78565 10.1333 8.99204Z"
                  fill="#191919"
                />
              </g>
              <defs>
                <clippath
                  id="clip0_511_22270"
                >
                  <rect
                    fill="white"
                    height="16"
                    transform="translate(0 0.5)"
                    width="16"
                  />
                </clippath>
              </defs>
            </svg>
            <p
              class="dropdown__contacts__text"
            >
              Mon-Sun: 11:00 – 20:00
            </p>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="dropdown"
    >
      <div
        class="dropdown__top"
      >
        <div />
        <h4
          class="dropdown__title"
        >
          close
        </h4>
        <svg
          fill="none"
          height="16"
          viewBox="0 0 16 16"
          width="16"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clip-rule="evenodd"
            d="M7.12587 8L2 2.87413L2.87413 2L8 7.12587L13.1259 2L14 2.87413L8.87413 8L14 13.1259L13.1259 14L8 8.87413L2.87413 14L2 13.1259L7.12587 8Z"
            fill="white"
            fill-rule="evenodd"
          />
        </svg>
      </div>
      <div
        class="dropdown__search"
      >
        <svg
          fill="none"
          height="33"
          viewBox="0 0 32 33"
          width="32"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M22.1553 24.5414L27.8647 30.2521L29.7513 28.3654L24.0407 22.6561C25.7436 20.5317 26.6699 17.8894 26.666 15.1667C26.666 8.54275 21.29 3.16675 14.666 3.16675C8.04202 3.16675 2.66602 8.54275 2.66602 15.1667C2.66602 21.7907 8.04202 27.1667 14.666 27.1667C17.3887 27.1706 20.031 26.2444 22.1553 24.5414ZM21.166 21.8667C19.4259 23.5589 17.0933 24.5039 14.666 24.5001C9.50868 24.5001 5.33268 20.3227 5.33268 15.1667C5.33268 10.0094 9.50868 5.83341 14.666 5.83341C19.822 5.83341 23.9993 10.0094 23.9993 15.1667C24.0032 17.594 23.0582 19.9266 21.366 21.6667L21.166 21.8667Z"
            fill="#191919"
          />
        </svg>
        <form>
          <input
            class="dropdown__search__title"
            name="search"
            placeholder="SEARCH"
            type="text"
            value=""
          />
        </form>
      </div>
      <nav
        class="dropdown__menu"
      >
        <a
          class="dropdown__list"
          href="/catalogue?status=sell"
        >
          sale
        </a>
        <a
          class="dropdown__list"
          href="/catalogue?status=new"
        >
          new
        </a>
        <a
          class="dropdown__list"
          href="/catalogue"
        >
          catalogue
        </a>
        <a
          class="dropdown__list"
          href="/shops/kharkiv"
        >
          shops
        </a>
        <a
          class="dropdown__list"
          href="/about-us"
        >
          about us
        </a>
      </nav>
      <div
        class="dropdown__contacts"
      >
        <div
          class="dropdown__contacts__item"
        >
          <svg
            fill="none"
            height="17"
            viewBox="0 0 16 17"
            width="16"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              clip-path="url(#clip0_511_22277)"
            >
              <path
                d="M15.5261 2.3856C15.7864 2.12525 15.7864 1.70314 15.5261 1.44279L15.0547 0.971385C14.7943 0.711035 14.3722 0.711036 14.1119 0.971385L11.2834 3.79983L10.1049 2.62127C9.9142 2.43061 9.62746 2.37357 9.37833 2.47676C9.1292 2.57995 8.9668 2.82303 8.9668 3.09268V6.86391C8.9668 7.23213 9.26526 7.5306 9.63346 7.5306H13.4047C13.6743 7.5306 13.9175 7.36813 14.0206 7.11903C14.1238 6.86992 14.0668 6.58317 13.8761 6.39251L12.6977 5.21404L15.5261 2.3856Z"
                fill="#191919"
              />
              <path
                d="M1.33791 3.52769C1.27065 2.57483 2.07752 1.83325 3.03079 1.83325H5.07135C5.56069 1.83325 6.00203 2.12809 6.19023 2.58072L7.04351 4.63289C7.25138 5.1328 7.10385 5.71021 6.68185 6.04853L6.14994 6.47495C5.91257 6.66524 5.83187 6.99338 5.97261 7.26307C6.69591 8.64934 7.76491 9.82581 9.06491 10.6777C9.33931 10.8575 9.70178 10.786 9.90651 10.5295L10.4608 9.83527C10.7984 9.41241 11.3746 9.26454 11.8734 9.47287L13.9214 10.3279C14.3731 10.5165 14.6673 10.9588 14.6673 11.4491V12.1577H14.6399V13.4665C14.6399 14.4217 13.8999 15.2303 12.949 15.1626C6.86871 14.7301 1.76896 9.63441 1.33791 3.52769Z"
                fill="#191919"
              />
            </g>
            <defs>
              <clippath
                id="clip0_511_22277"
              >
                <rect
                  fill="white"
                  height="16"
                  transform="translate(0 0.5)"
                  width="16"
                />
              </clippath>
            </defs>
          </svg>
          <p
            class="dropdown__contacts__text"
          >
            Call me out
          </p>
        </div>
        <div
          class="dropdown__contacts__item"
        >
          <svg
            fill="none"
            height="17"
            viewBox="0 0 16 17"
            width="16"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M1.33791 3.52769C1.27065 2.57483 2.07752 1.83325 3.03079 1.83325H5.07135C5.56069 1.83325 6.00203 2.12809 6.19023 2.58072L7.04351 4.63289C7.25138 5.1328 7.10385 5.71021 6.68185 6.04853L6.14994 6.47495C5.91257 6.66524 5.83187 6.99338 5.97261 7.26307C6.69591 8.64934 7.76491 9.82581 9.06491 10.6777C9.33931 10.8575 9.70178 10.786 9.90651 10.5295L10.4608 9.83527C10.7984 9.41241 11.3746 9.26454 11.8734 9.47287L13.9214 10.3279C14.3731 10.5165 14.6673 10.9588 14.6673 11.4491V12.1577H14.6399V13.4665C14.6399 14.4217 13.8999 15.2303 12.949 15.1626C6.86871 14.7301 1.76896 9.63441 1.33791 3.52769Z"
              fill="#191919"
            />
          </svg>
          <a
            class="dropdown__contacts__text"
            href="tel:+380505000096"
          >
            +380(50) 500 00 96
          </a>
        </div>
        <div
          class="dropdown__contacts__item"
        >
          <svg
            fill="none"
            height="17"
            viewBox="0 0 16 17"
            width="16"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              clip-path="url(#clip0_511_22270)"
            >
              <path
                d="M13.9045 6.25251C14.1618 6.25251 14.4036 6.15238 14.5855 5.97049C14.7674 5.78856 14.8676 5.54677 14.8676 5.28952C14.8676 5.03227 14.7674 4.79048 14.5855 4.60859L13.6322 3.65529C13.4503 3.47336 13.2085 3.37323 12.9513 3.37323C12.694 3.37323 12.4522 3.47336 12.2703 3.65529C11.9251 4.0005 11.8976 4.54469 12.1872 4.92196L11.9604 5.13448C11.1502 4.48702 10.1796 4.03234 9.11842 3.84118V3.35659C9.88137 3.328 10.4933 2.69877 10.4933 1.92898C10.4933 1.14103 9.85229 0.5 9.06435 0.5H6.90449C6.11655 0.5 5.47552 1.14103 5.47552 1.92898C5.47552 2.69877 6.08747 3.328 6.85042 3.35659V3.84118C5.77996 4.03404 4.8017 4.49504 3.98725 5.15154L3.79205 4.94874C4.10406 4.57086 4.08345 4.00878 3.73003 3.65529C3.54814 3.47336 3.30628 3.37323 3.04907 3.37323C2.79185 3.37323 2.55003 3.47336 2.36814 3.65529L1.41487 4.60859C1.23298 4.79048 1.13281 5.03227 1.13281 5.28952C1.13281 5.54677 1.23298 5.78856 1.41487 5.97045C1.59676 6.15238 1.83862 6.25251 2.09584 6.25251C2.31346 6.25251 2.51997 6.18055 2.68858 6.04831L2.90129 6.26934C2.08797 7.34025 1.60451 8.67472 1.60451 10.1201C1.60451 13.638 4.46655 16.5 7.98444 16.5C11.5023 16.5 14.3643 13.638 14.3643 10.1201C14.3643 8.66466 13.8741 7.32172 13.0505 6.24703L13.2853 6.02691C13.4586 6.17287 13.6754 6.25251 13.9045 6.25251ZM13.2299 10.1201C13.2299 13.0124 10.8768 15.3656 7.98444 15.3656C5.09208 15.3656 2.73895 13.0124 2.73895 10.1201C2.73895 7.22775 5.09208 4.87462 7.98444 4.87462C10.8768 4.87466 13.2299 7.22775 13.2299 10.1201Z"
                fill="#191919"
              />
              <path
                d="M7.98418 6.07544C5.75391 6.07544 3.93945 7.8899 3.93945 10.1202C3.93945 12.3504 5.75391 14.1649 7.98418 14.1649C10.2144 14.1649 12.0289 12.3504 12.0289 10.1202C12.0289 7.8899 10.2144 6.07544 7.98418 6.07544ZM10.1333 8.99204L8.79761 10.1618C8.78774 10.3564 8.70939 10.5482 8.56078 10.6968C8.24234 11.0152 7.72602 11.0152 7.40758 10.6968C7.08914 10.3783 7.08914 9.862 7.40758 9.54357C7.58376 9.36739 7.82044 9.28911 8.05073 9.30791L9.38585 8.13864C9.62155 7.93229 9.97984 7.95596 10.1863 8.19162C10.3927 8.42732 10.3689 8.78565 10.1333 8.99204Z"
                fill="#191919"
              />
            </g>
            <defs>
              <clippath
                id="clip0_511_22270"
              >
                <rect
                  fill="white"
                  height="16"
                  transform="translate(0 0.5)"
                  width="16"
                />
              </clippath>
            </defs>
          </svg>
          <p
            class="dropdown__contacts__text"
          >
            Mon-Sun: 11:00 – 20:00
          </p>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
